misk-aws2-dynamodb/build.gradle

```diff
@@ -3,6 +3,7 @@ dependencies {
 
   implementation dep.guice
   implementation project(':misk-aws')
+  implementation project(':misk-exceptions-dynamodb')
```

Comment:
I tried to find these exceptions in the aws2 SDK and couldn't find which Gradle dependency to include. Any ideas?